docs/docs/components/alert.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Alert
+sidebar_position: 5
+---
+
+<DocChip chip='shadow' />
+<DocChip chip='name' label="dwc-alert" />
+
+The `Alert` component in webforJ provides contextual feedback messages for users. It's a versatile way to display important information, warnings, or notifications in your app.
+
+Alerts help draw attention to key information without disrupting the user's workflow. They're perfect for system messages, form validation feedback, or status updates that need to be clearly visible but not intrusive.
+
+Here's an example of an alert component:
+
+<ComponentDemo 
+path='/webforj/alert?' 
+javaE='https://raw.githubusercontent.com/webforj/webforj-documentation/refs/heads/main/src/main/java/com/webforj/samples/views/alert/AlertView.java'
+height = '100px'
+/>
+
+## Dismissing alerts
+
+If you’d like to give users the option to dismiss the `Alert`, you can make it closable either by calling the `setClosable()` method.
+
+```java 
+Alert alert = new Alert("Heads up! This alert can be dismissed.");
+closableAlert.setClosable(true);
+```
+
+<ComponentDemo 
+path='/webforj/closablealert?' 
+javaE='https://raw.githubusercontent.com/webforj/webforj-documentation/refs/heads/main/src/main/java/com/webforj/samples/views/alert/ClosableAlertView.java'
+height = '125px'
+/>
+
+
+:::tip
+Closing the alert only hides it—it doesn’t destroy the component, so you can reuse it later if needed.
+:::
+
+### Close event
```

Comment:
Combine the close event documentation with the earlier “Dismissing Alerts” section. Update the `Dismissing Alerts` demo to show a primary-themed button only after the alert is dismissed. Then remove the separate close event demo, it’s a bit confusing and hard to follow.